### DISCUSSION TITLE
nodes react but no update for power and other

### DISCUSSION THREAD

**Discussion by peter-dtn** [zwave-js/zwave-js-ui#1338] [Q&A] [answered]
Hi Team, 

thanks again for the excellent software !

I did use the zwave2mqtt for long time but yesterday I decided to make the step to zwavejs2mqtt. After some time all seems to work but when switching ON a switch I don't get the power or other items automatically.

e.g. switch On a Fibaro wallplug with ~600Watt load in the control-panel is successful, but the measured power by the wall-plug stays at 0Watt. Only when pushing the "refresh" button in the control-panel I can manage to see the power.

I'm using version 5.0.1, around 25 nodes and all seems to have trouble with reading values.

Hope you can give me some direction..

**Comment by robertsLando** [comment, marked as answer]
Please check the association groups, if your device isn't reporting changes it could be because the lifeline association is not setted up correctly with the controller or could be a device issue. @AlCalzone

**Reply by AlCalzone** [reply]
To be specific, OZW used to make associations to node 1, endpoint 1. If that exists on any of your devices, remove it and add an association to node 1, endpoint 0 instead. Or no target endpoint if endpoint 0 is not possible.

**Comment by peter-dtn** [comment]
Many thanks, this was the hint to solve the issue ! What I did =>

For each node I performed "Advanced => Associations => Clear" followed by "Advanced => Interview"
I watched the traffic in the debug window and saw the node updating.

The nodes are working as before and I have the great feeling my Zwave network is under control again!

Thanks !
